2015-Fall/adrianbrink/nonFunkyCalculator/nonFunkyCalculator/ViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by aevitas on 01/09/15.
+//  Copyright (c) 2015 aevitas. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var display: UILabel!
+    
+    var userIsTyping: Bool = false
```

Comment:
You can omit the explicit type here as it is clear from the context, i.e. `var userIsTyping = false`